File: 0168.excel-sheet-column-title/excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int n) {
        String[] a = {"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"};
        String ans = "";
        while(n > 0)
        {
            n--;
            ans = a[n % 26] + ans;
            n /= 26;
        }
        return ans;
        /* StringBuilder ans = new StringBuilder();
        while (n > 0)
        {
            n--;
            ans.insert(0, (char)('A' + n % 26));
            n /= 26;
        }
        return ans.toString();*/
    }
}

File: 0168.excel-sheet-column-title/excel-sheet-column-title.cs
public class Solution {
    public string ConvertToTitle(int n) {
         string[] a = new string[26] {"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"};
        string ans = "";
        n--;
        while(n / 26 != 0)
        {
            ans = a[n % 26] + ans;
            n /= 26;
            n--;
        }
        ans = a[n % 26] + ans;
        return ans;
    }
}